### PR TITLE
Issue #29. Header: a start.

### PR DESCRIPTION
**Pull request by rlr**
I would like to merge early and often. Or at least get feedback early and often so I don't go down too far on the wrong road.

So here's a start. I have some tests but I just realized karma/mocha/whateverrunsthem isn't hooked up yet so I stashed them.

@k88hudson

### CODE REVIEW

File: data/content/activity-streams.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang=en-us>
   <head>
     <meta charset="utf8">
-    <link rel="stylsheet" href="http://localhost:1963/main.css" />
```
Comment (k88hudson) [inline data/content/activity-streams.html]: lol oops

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+const React = require("react");
+
+const Header = React.createClass({
+  render() {
+    const props = this.props;
+    return (<header className="head">
+      <section className="nav">
+        Home
+      </section>
+      <section className="filters">
```
Comment (k88hudson) [inline src/components/Header/Header.js]: According to the product requirements we can actually omit this i think

File: src/components/Header/Header.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
+.head {
+  align-items: center;
+  background: #f5f6f7;
```
Comment (k88hudson) [inline src/components/Header/Header.scss]: Should be white (except for the menu thingy on the left, which we can add later)

File: src/components/Header/Header.scss [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  border-bottom: solid 1px #dfe0e1;
+  display: flex;
+  line-height: 30px;
```
Comment (k88hudson) [inline src/components/Header/Header.scss]: this shouldn't be necessary if you have `align-items: center`

Comment (k88hudson) [inline src/components/Header/Header.scss]: make the `height 50px;` instead, so we can have stuff be centered and everyhting

File: src/components/Header/Header.scss [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  padding: 5px;
```
Comment (k88hudson) [inline src/components/Header/Header.scss]: make this `0 20px`; you don't need the vertical padding if you have `align-items: center`, and eventually we'll need the menu on the left to touch the top/bottom

File: src/components/Header/Header.scss [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+  section {
+    display: flex;
```
Comment (k88hudson) [inline src/components/Header/Header.scss]: throw `align-items: center` on this too

File: src/components/Header/Header.scss [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+    &.nav {
+      border-right: solid 1px #dfe0e1;
```
Comment (k88hudson) [inline src/components/Header/Header.scss]: this should have `background: #F3F4F6`, no border necessary

also make this `height: 100%`

Comment (rlr) [inline src/components/Header/Header.scss]: are the colors specified somewhere? my eyedropper failed me LOL

**Comment by rlr** [comment]
ok, I went through the feedback and added a first test ^^

**Comment by k88hudson** [comment]
R+

**Pull request closed without merging**